src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -423,34 +416,24 @@
     <WriteItemsToJson JsonFileName="%(TestListFile.BuildCompleteJson)" Items="@(BuildComplete)" />
   </Target>
 
-  <!-- send completion events -->
+  <!-- Send completion event.  All start messages are now sent to Helix API; Event Hub direct sending is now deprecated. -->
   <Target Name="SendCompletionEvent"
           AfterTargets="WriteCompletionEvent"
           Inputs="%(TestListFile.BuildCompleteJson)"
           Outputs="%(TestListFile.HelixJobUploadCompletePath)"
           Condition="'$(SkipNotifyEvent)' != 'true'">
-    <SendToEventHub Condition="'$(HelixApiAccessKey)' == ''"
-      EventHubPath="$(EventHubPath)"
-      EventData="%(TestListFile.BuildCompleteJson)"
-      EventHubSharedAccessKeyName="$(EventHubSharedAccessKeyName)"
-      EventHubSharedAccessKey="$(EventHubSharedAccessKey)"/>
+    
+    <Error Condition="'$(HelixApiAccessKey)' == ''" Text="Please provide a value for the 'HelixApiAccessKey' property." />
```

Comment:
I'd fix up the condition in VerifyInputs so that it's applicable to both cases, that way you don't have to wait for the build to eventually fail to realize you missed a property.